src/dotnet/commands/dotnet-clean/Program.cs

```diff
@@ -17,20 +17,28 @@ public static int Run(string[] args)
             CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
             {
                 Name = "dotnet clean",
-                FullName = ".NET Clean Command",
-                Description = "Command to clean previously generated build outputs.",
+                FullName = LocalizableStrings.AppFullName,
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
             app.HelpOption("-h|--help");
 
-            CommandArgument projectArgument = app.Argument("<PROJECT>",
-                "The MSBuild project file to build. If a project file is not specified," +
-                " MSBuild searches the current working directory for a file that has a file extension that ends in `proj` and uses that file.");
+            CommandArgument projectArgument = app.Argument($"<{LocalizableStrings.CmdArgProject}>",
```

Comment:
Move parameters to their own lines when you move any to their own line. You left this one and a couple others. Do a pass and see other cases that may apply.